web-ui/src/main/resources/catalog/components/edit/onlinesrc/OnlineSrcService.js

```diff
@@ -261,18 +262,31 @@
           }
 
           linksAndRelatedPromises.push(
-            $http.get(apiPrefix + "/related?type=" + relatedTypes.join("&type="), {
-              headers: {
-                Accept: "application/json"
+            $http.get(
+              apiPrefix + "/related?type=" + relatedTypes.join("&type=") + !isApproved
+                ? "&approved=false"
+                : "",
+              {
+                headers: {
+                  Accept: "application/json"
+                }
               }
-            })
+            )
           );
           linksAndRelatedPromises.push(
-            $http.get(apiPrefix + "/associated?type=" + associatedTypes.join("&type="), {
-              headers: {
-                Accept: "application/json"
+            $http.get(
+              apiPrefix +
+                "/associated?type=" +
+                associatedTypes.join("&type=") +
+                !isApproved
+                ? "&approved=false"
+                : "",
```

Comment:
```suggestion
                (!isApproved
                ? "&approved=false"
                : ""),
```

The code was wrong, sending only as URL the value `"&approved=false"` and causing the online resources not displayed in the metadata editor.

I'm not totally sure if for the associated resources this logic is required.